### PR TITLE
Several enhancements

### PR DESCRIPTION
**Pull request by NewbProgrammer101**
fill_chest.lua is a work in progress.

### CODE REVIEW

**Comment by NewbProgrammer101** [comment]
This is very WIP because still not sure if it spawns in Iron Hills specifically because I haven't found one yet.

**Comment by NewbProgrammer101** [comment]
I finally found a geomorph directly below the Iron Hills biome thanks to the geomorph() call in lottmapgen.biomes(11).
The coordinates are about y < -250.

**Comment by NewbProgrammer101** [comment]
Minor changes with the spawning.

**Comment by NewbProgrammer101** [comment]
I added the latest version of WorldEdit compatible with 0.5.0 Minetest

**Comment by NewbProgrammer101** [comment]
Changed mapgen.lua

**Comment by NewbProgrammer101** [comment]
I'm done with attempting to make geomoria spawn in iron hills.

Feel free to make corrections.

**Comment by NewbProgrammer101** [comment]
Expect me to constantly update trader_goods.lua

**Comment by NewbProgrammer101** [comment]
Made chairs sittable.

**Comment by NewbProgrammer101** [comment]
Just merge this PR already. I've already spent many hours on this.

**Comment by NewbProgrammer101** [comment]
You may now sleep in beds upon rightclick.

**Comment by NewbProgrammer101** [comment]
Question: Why am I not getting any feedback?
I don't know whether geomoria is compatible with lottmapgen after all these commits. Is it?
I'm sure you'll merge my commits on making the chairs sittable and beds sleepable.
Anything else I should do?

**Comment by NewbProgrammer101** [comment]
To test my fork of your subgame just git clone my repository into your .minetest/games folder and test it.

Hello?

**Comment by fishyWET** [comment]
Hi, we are still thinking about it.
The issue is that, the current LOTT will be discontinued.
A newer rebased, rebalanced, cleaner version will replace it, and the current version will be the last version to support the old LOTT, before the new one basically breaks everything.

We are unsure whether to add in something so major that might have bugs, as we want the final version to be as bugless as possible - as it will be the last one to support anyone who can't migrate over to the new LOTT - server owners with a half developed world etc.

**Comment by fishyWET** [comment]
Also, it would be much better if you can split the changes into 4 different pull requests - beds, WE update, trader convo, geomoria.
The 4 changes are very much different things, due to the reason above, geomoria is primarily blocking the other changes from being merged.

You can create separate branches in your repo, each of which is focused on a single change.

**Comment by NewbProgrammer101** [comment]
I removed geomoria from the mod collection for the time being. I'll work on it further. 

You can merge my extra stuff.

File: mods/lottblocks/init.lua [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-dofile(minetest.get_modpath("lottblocks").."/music.lua") --This has musical insturments, and the music they play.
-dofile(minetest.get_modpath("lottblocks").."/decoration.lua") --This has various decoration nodes, by catninja.
-dofile(minetest.get_modpath("lottblocks").."/chests.lua")
-dofile(minetest.get_modpath("lottblocks").."/lamps.lua")
-dofile(minetest.get_modpath("lottblocks").."/banners.lua")
-dofile(minetest.get_modpath("lottblocks").."/doors.lua")
-dofile(minetest.get_modpath("lottblocks").."/palantiri.lua")
-dofile(minetest.get_modpath("lottblocks").."/crafting.lua")
-dofile(minetest.get_modpath("lottblocks").."/guides.lua")
+lottblocks.formspec = "size[8,15;true]" ..
```
Comment (fishyWET) [inline mods/lottblocks/init.lua]: Move under beds.lua
Localize and rename to bed_formspec

**Comment by fishyWET** [comment]
Also you appeared to have removed WE, WE is needed for the buildings to generate.

**Comment by NewbProgrammer101** [comment]
I thought I only removed geomoria, I didn't know I also accidentally removed WE.

I will get WE 1.2 stable back here.

**Comment by NewbProgrammer101** [comment]
Satisfied the change requests.

**Pull request merged**